src/__tests__/InputField.js

```diff
@@ -24,6 +24,105 @@ describe('InputField', () => {
     )
     console.error.mockRestore()
   })
+  test('it renders using FormField alias', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Input id="with-left-icon" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select ', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Select id="with-no-options" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select and Icon', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="email" />
+          <Select id="with-left-icon" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select and Options', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Select id="with-options">
+            <option>Option 1</option>
+            <option>Option 2</option>
+            <option>Option 3</option>
+          </Select>
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select icons on both sides', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="key" />
+          <Select id="with-options">
+            <option>Option 1</option>
+            <option>Option 2</option>
+            <option>Option 3</option>
+          </Select>
+          <Icon name="email" />
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select icons on the left side', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
+          <Icon name="key" />
+          <Select id="with-options">
+            <option>Option 1</option>
+            <option>Option 2</option>
+            <option>Option 3</option>
+          </Select>
+        </FormField>
+      )
+      .toJSON()
+    expect(json).toMatchSnapshot()
+  })
+
+  test('it renders using FormField alias with Select icons on the right side', () => {
+    const json = renderer
+      .create(
+        <FormField onChange={() => {}}>
```

Comment:
Is the `onChange` prop required in the `FormField` composition? If yes and it's being used in any test case, perhaps we can think of ways to test `onChange` instead of the empty function?

Another thought is that, perhaps, the PR can keep it simple and finds way to not to require `onChange` in the `FormField` composition? Seems like your code coverage is 100% for `FormField`, and it's green for CI. So probably another open issue can explore how to test the `onChange` in design system unit test, or similar issue topic? (just thoughts...)

Reply:
That should have been fixed in #286 – I think this onChange can be removed for most tests, but we'd want a single test to ensure it works for backwards API compatibility